ClosedXML/Excel/XLWorkbook_Save.cs

```diff
@@ -304,7 +304,7 @@ private void CreateParts(SpreadsheetDocument document, SaveOptions options)
                     GenerateVmlDrawingPartContent(vmlDrawingPart, worksheet, context);
                 }
 
-                GenerateWorksheetPartContent(worksheetPart, worksheet, options.EvaluateFormulasBeforeSaving, context);
+                GenerateWorksheetPartContent(worksheetPart, worksheet, options.EvaluateFormulasBeforeSaving, options.ConsolidateConditionalFormatRanges, options.ConsolidateDataValidationRanges, context);
```

Comment:
Maybe simply pass `SaveOptions` to the method? For one flag it was OK to pass a boolean, but for three it looks weird.

Reply:
adjusted according to your comment